task-12/04-ansible/database-servers/mysql-yum-repo.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+ sudo rpm --import https://repo.mysql.com/RPM-GPG-KEY-mysql-2022
+ wget http://dev.mysql.com/get/mysql57-community-release-el7-8.noarch.rpm
+ sudo yum localinstall -y mysql57-community-release-el7-8.noarch.rpm
+ sudo yum install -y mysql-community-server
+ sudo systemctl start mysqld
```

Comment:
you can use module instead of this inside your task
https://docs.ansible.com/ansible/latest/collections/ansible/builtin/dnf_module.html